kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnector.java

```diff
@@ -85,4 +92,38 @@ public ConfigDef config() {
     return S3SinkConnectorConfig.getConfig();
   }
 
+  public boolean checkBucketExists(String bucketName) {
```

Comment:
The method has been made public for mocking purposes. Making it private was giving errors. Also, mocking private methods is not considered good practice.

Reply:
```suggestion
  protected boolean checkBucketExists(String bucketName) {
```